examples/jsm/controls/OrbitControls.js

```diff
@@ -166,7 +166,7 @@ class OrbitControls extends EventDispatcher {
 
 			return function update() {
 
-				const position = scope.object.position;
+				const position = getPosition( scope.object, scope.object.position );
```

Comment:
I think it's more clear to define something like `_position` in the closure scope above and reuse it in `update()`.